apps/src/code-studio/components/DatasetPicker.jsx

```diff
@@ -55,14 +54,13 @@ export default class DatasetPicker extends React.Component {
 
   render() {
     const datasetEntries = datasetLibrary.datasets.map(d => {
-      const choose = this.chooseAsset.bind(this, d.name, d.url);
       return (
```

Comment:
nit: now this can just be:
```
    const datasetEntries = datasetLibrary.datasets.map(d => (
      <DatasetListEntry
        ...
      />
    ));
```